Jenkinsfile_CNP

```diff
@@ -0,0 +1,29 @@
+#!groovy
+
+@Library("Infrastructure")
+
+def type = "nodejs"
+def product = "em"
+def component = "icp"
+
+def secrets = [
```

Comment:
Can't you avoid using s2s secret from s2s vault. I am guessing you have plans to copy to ur vault anyway?